com.cubrid.common.ui/src/com/cubrid/common/ui/common/preference/JdbcManageComposite.java

```diff
@@ -295,20 +295,15 @@ public void run() {
 					if (taskExec.isSuccess()) {
 						List<String> downloadedList = task.getDriverList();
 						if (downloadedList != null && downloadedList.size() > 0) {
-							downloadedFiles = downloadedList.toString();
+							downloadedFiles = downloadedList.toString().replace(',', '\n');
 							for (int i = 0; i < jdbcList.size(); i++) {
```

Comment:
This fix is good to see me. But I have one issue like below.
When String variable `downloadedFiles` printed in informationBox, Brackets also printed too. 
![bracket1](https://cloud.githubusercontent.com/assets/16603187/20995304/78a274a0-bd39-11e6-94f3-787db0522b2a.png)
![bracket2](https://cloud.githubusercontent.com/assets/16603187/20995307/7a6e6410-bd39-11e6-99ec-4a616a18097a.png)

I think that reason why is converting `List` to `String` using `toString()`.
If below code added to 299 line, that issue will be resolved.
```java
downloadedFiles = downloadedFiles.substring(1, downloadedFiles.length() - 1);
```